Sources/RunnerLib/SPMDanger.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+public struct SPMDanger {
+    private static let dangerDepsPrefix = "DangerDeps"
+    public static let buildFolder = ".build/debug"
+    public let depsLibName: String
+
+    public init?(packagePath: String = "Package.swift") {
+        let packageContent = (try? String(contentsOfFile: packagePath)) ?? ""
+
+        let regex = try? NSRegularExpression(pattern: "\\.library\\(name:[\\ ]?\"(\(SPMDanger.dangerDepsPrefix)[A-Za-z]*)", options: .allowCommentsAndWhitespace)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/RunnerLib/SPMDanger.swift;
  Line: 11;
-->

- :warning: Line should be 120 characters or less: currently 161 characters (`line_length`)